fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/FedoraOCFLMapping.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.ocfl.impl;
+
+/**
+ * A data structure that linking parent fedora resource identifiers to its corresponding OCFL Object Id.
+ *
+ * @author dbernstein
+ */
+public class FedoraOCFLMapping {
+    private String parentFedoraResourceId;
+    private String ocflObjectId;
+
+    /**
+     * Default constructor
+     * @param parentFedoraResourceId The parent fedora resource identifier
+     * @param ocflObjectId The OCFL Object identitifer
+     */
+    public FedoraOCFLMapping(final String parentFedoraResourceId, final String ocflObjectId){
+        this.parentFedoraResourceId = parentFedoraResourceId;
+        this.ocflObjectId = ocflObjectId;
+    }
+
+    /**
+     * Retrieve the parent fedora resource identifier associated with the OCFL Object Id.
+     * It is a "parent" identifier because it refers to the Archival Group identifier for
```

Comment:
similar to my comment a few files back, there might be other wrapped objects like webacs. Maybe it can just be "the id for the fedora resource which represents this ocfl object"

Reply:
done.